### PR TITLE
patches wrt MC's internal review

### PR DESCRIPTION
**Pull request by aaarcher-usgs**
Patch to make some minor changes:

- Couple errors in the text
- Add a reference to WDFN dashboard (per FSP review)
- Update methods figures to better match some requests from FSP reviewers
- Add notes to timeline about when more stream gages were added to the dataset(s)

Bigger changes:

- there is now a 1970s inset map that shows up during that pseudo-major drought event on desktop
- scroll triggers were modified slightly on desktop so that the inset map uses the same scroll triggers as the quote and narration

### CODE REVIEW

**Comment by cnell-usgs** [comment]
Love this arrow 
image

File: src/assets/images/severity_explainer.png
Comment (cnell-usgs) [inline src/assets/images/severity_explainer.png:1]: What if there was a highlight color used on these to emphasize what the measurement is? I see the severity area is darker and the arrows, but it could be more pronounced.

File: src/assets/text/droughtNarrations_desktop.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ export default {
...
         end_date: '1928-12-31', // Date marking end of annotation. Determines y trigger end position on deskop
-        text: 'This timeline shows the 2,000 most severe streamflow drought events at USGS streamgages over 100 years (<a href="https://labs.waterdata.usgs.gov/visualizations/what-is-drought/index.html#/" target="_blank">What is Streamflow Drought?</a>). Each bar shows the duration of drought at a single gage so that the taller and darker the bar, the longer the drought event. When multiple streamgages experienced drought conditions simultaneously, the chart is wider.', // Narrative text
+        text: 'This timeline shows the 2,000 most severe streamflow drought events at USGS streamgages over 100 years (<a href="https://labs.waterdata.usgs.gov/visualizations/what-is-drought/index.html#/" target="_blank">What is Streamflow Drought?</a>). Each bar shows the duration of drought at a single gage so that the taller and darker the bar, the longer the drought event. The map above shows streamgages that had one of the 2,000 most severe droughts, with gages in drought highlighted in red for each major drought period.', // Narrative text
```
Comment (cnell-usgs) [inline src/assets/text/droughtNarrations_desktop.js:8]:
```suggestion
        text: 'This timeline shows the 2,000 most severe streamflow drought events at USGS streamgages over 100 years (<a href="https://labs.waterdata.usgs.gov/visualizations/what-is-drought/index.html#/" target="_blank">What is Streamflow Drought?</a>). Each bar shows the duration of drought at a single gage. The taller and darker the bar, the longer the drought event was. The map above shows all of the streamgage locations, with red used to show gages in drought for each major drought period.', // Narrative text
```

**Comment by cnell-usgs** [comment]
Annotations:
1. The "Time" with arrow at the top could be placed closer to the bottom of the page to signal that the page scrolls. If this is going to be difficult to do with different screen sizes, don't bother.
2. "Wider = more simultaneous droughts". Suggest rewording: "Number of gages in drought" 
3. No colorbar?

**Review by cnell-usgs — APPROVED** [review]
Some minor text suggestions to review before merging.

**Comment by aaarcher-usgs** [comment]
Optional landing page changes: What do you think @cnell-usgs?

image

**Comment by cnell-usgs** [comment]
Looks good

File: src/assets/images/severity_explainer.png
Comment (aaarcher-usgs) [reply src/assets/images/severity_explainer.png:1]: Good idea!
